modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/fieldSupport.js

```diff
@@ -353,25 +350,19 @@ export const normalizeSettingsContextPages = (
 				});
 			}
 
-			const newInstanceId = generateInstanceId(8);
+			const instanceId = generateInstanceId(8);
 
 			if (field.type === 'rich_text' && field.editorConfig) {
-				field = {
```

Comment:
This code will go back, because of the change in `updateEditorConfigInstanceId` for a pure function, @miltonmc.